### PR TITLE
Updates From July 2nd

### PR DESCRIPTION
**Pull request by laumurph**
Implements all the changes that were requested in red except the request in the python modules chapter. That change is implemented in the pull request specific to the python module chapter.

### CODE REVIEW

File: conf.py [hunk 1]
```diff
@@ -31,10 +31,6 @@
 
 #,'runestone.video','runestone.reveal','runestone.poll','runestone.tabbedStuff','runestone.disqus','runestone.codelens','runestone.activecode', 'runestone.assess', 'runestone.animation','runestone.meta', 'runestone.parsons', 'runestone.blockly', 'runestone.livecode']
 
-extension_exclusions = ['runestone.matrixeq', 'runestone.webgldemo']
-for ext in extension_exclusions:
-    extensions.remove(ext)
```
Comment (presnick) [inline conf.py]: Why this change?

File: old_conf.py [hunk 2]
```diff
@@ -0,0 +1,237 @@
+# -*- coding: utf-8 -*-
```
Comment (presnick) [inline old_conf.py]: Why create this? git keeps track of all our old versions of conf.py anyway.

File: _sources/Sequences/StringsandLists.rst [hunk 3]
```diff
@@ -65,6 +62,35 @@ another list.
...
+parentheses () instead of square brackets [].
+
+One way to create a tuple is to write an expression, enclosed in parentheses,
+that consists of multiple other expressions, separated by commas.
+
+.. sourcecode:: python
+
+    julia = ("Julia", "Roberts", 1967, "Duplicity", 2009, "Actress", "Atlanta, Georgia")
+
+Tuples are useful for representing what other languages often call *records* --- some related information that belongs
```
Comment (presnick) [inline _sources/Sequences/StringsandLists.rst]: Please omit this paragraph. Nothing special about tuples for storing records; could be lists just as well.

Please add:
The key difference between lists and tuples is that a tuple is *immutable*, meaning that its contents can't be changed after the tuple is created. We will examine the mutability of lists in detail in the chapter on `Mutability`_.

File: _sources/Iteration/TraversalandtheforLoopByIndex.rst [hunk 4]
```diff
@@ -27,29 +27,10 @@ allows us to loop through something and have an automatic counter.
        print(counter, item)
```
Comment (presnick) [inline _sources/Iteration/TraversalandtheforLoopByIndex.rst]: I think we can make it clearer that we're using enumerate as a building block to help their understanding here. I'll do a proposed rewrite as a separate pull request.

File: conf.py (hunk 1 above)
Comment (bnmnetp) [reply conf.py]: I would also vote to keep this change.    We have so many js files to load that any little things like this we can do to reduce the number is a good thing.

Comment (laumurph) [reply conf.py]: I meant to remove this file and rename the 'old_conf.py' file back to 'conf.py'. I had fetched and merged the latest updates from this repo to my own, but the changes in the conf.py file meant that I wasn't able to properly build the textbook. This is an old version of my conf.py file that still works for me.

Comment (bnmnetp) [reply conf.py]: @laumurph are you using the latest version of Runestone Components?  If its not building for you, please make an issue with the problem so I can track it down.
